Makefile

```diff
@@ -1,26 +1,57 @@
-.PHONY: help start stop test shell flush loaddata
 .DEFAULT_GOAL := help
 
+.PHONY: help
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort -k 1,1 | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
+.PHONY: start
 start: ## Start the development server
 	@docker-compose up -d --build
 
+.PHONY: stop
 stop: ## Stop the development server
 	@docker-compose stop
 
+.PHONY: lint
+lint: ## Lint the project
+	@docker-compose exec backend sh -c "poetry run black --check . && poetry run flake8"
+
+.PHONY: format-code
+format-code: ## Format the backend code
+	@docker-compose exec backend sh -c "poetry run black . && poetry run isort ."
+
+.PHONY: test
 test: ## Test the project
-	@docker-compose exec backend sh -c "poetry run black --check . && poetry run flake8 && poetry run pytest --no-cov-on-fail --cov"
+	@docker-compose exec backend poetry run pytest --no-cov-on-fail --cov
```

Comment:
YMMV, but I like it if `make test` also lints the code.

Reply:
Ok, changed it back